Formula/osslsigncode.rb

```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "2549903650fdd0189c1083a177af1c789364758080828b8ea6cf46c376ed6d65"
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "openssl@1.1"
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
```

Comment:
Can we report this upstream too?

Reply:
yeah, this was reported already

Reply:
Can you add a link to the upstream report in the comment?

Reply:
Ping?

Reply:
yeah, let me also file a separate report on the linker issue